tests/pom.xml

```diff
@@ -97,6 +97,16 @@
       <version>1.0.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.tyrus</groupId>
+      <artifactId>tyrus-client</artifactId>
+      <version>1.1</version>
```

Comment:
Do we need this or just for tests? If the latter, shouldn't it be in test scope?